_posts/2020-08-13-convertextract-app.md

```diff
@@ -0,0 +1,98 @@
+So, I added mappings in the [g2p library](https://github.com/roedoejet/g2p) 
+and a parser for ELAN in the Convertextract library. Aidan Pine then turned Convertextract into an application!
+
+# How to use the new Convertextract application for 'quality control' of ELAN annotations
+
+[Convertextract](https://github.com/roedoejet/convertextract), created by Aidan Pine, is a `python` library which extracts text data and finds/replaces specific text based on arbitrary correspondences. 
+Until now, only basic CLI (Command Line Interface) was supported. Using Convertextract in the CLI allowed the user to convert a file based on pre-existing Mappings in the [g2p library](https://github.com/roedoejet/g2p) or based on a custom mapping. However, the downside is that some programming knowledge is needed to use the CLI.
+The latest update now includes a GUI (Graphical User Interface) in the form of an application (for Mac **only**). The app makes Convertextract more accessible for non-programmers.
+
+### 1. G2P mapping
+G2P mappings (i.e. arbitrary conversions between an input and an output) can be created manually as a `.csv` or `.json` file, or with the [g2p studio app](g2p-studio.herokuapp.com).
+Many languages are already supported. See the [g2p repository](https://github.com/roedoejet/g2p) on Github for detailed documentation on the G2P library.
+
+### 2. Language data
+
+You language data must be in one of the supported file formats. The most recent addition is `.eaf` files, which allows ELAN annotations to be parsed and converted. For a full list of supported file types click [here](https://github.com/roedoejet/convertextract).
+
+### 3. Convertextract application
+
+#### Installation
+To download the app: [https://github.com/roedoejet/convertextract/releases](https://github.com/roedoejet/convertextract/releases).
+
+In your downloads folder, find the .zip file and double click on it to unzip.
+- Downloads>**convertextract**
+
+Right-click on the application in the dist folder and select *Open*.
+- Downloads>convertextract>dist>**Convertextract**
+
+*Note: If you try to double click to open the app, you will get a security message. Right-clicking to open will allow you to override the security message.*
+
+#### Using the app
+
+This is what the app looks like when you open it.
+
+![]([https://raw.githubusercontent.com/roedoejet/mothertongues-blog/assets/images/gui.png])
+
+All you have to do is add your language data, choose the encoding, and pick your g2p mapping!
+The output will be exported as a copy of the input file + *_converted.ext* in the filename.
+
+##### Example case
```

Comment:
This example case would be far more useful after the **motivation** section. That way the reader doesn't have to go through the entire post before being able to visualize the goal.